src/amo/components/Errors/ServerError/index.js

```diff
@@ -28,23 +27,21 @@ export class ServerErrorBase extends React.Component {
     /* eslint-disable react/no-danger */
     return (
       <NestedStatus code={500}>
-        <Page showWrongPlatformWarning={false}>
-          <Card
-            className="ErrorPage ServerError"
-            header={i18n.gettext('Server Error')}
-          >
-            <p>
-              {i18n.gettext(`
+        <Card
+          className="ErrorPage ServerError"
```

Comment:
Eesh, I just noticed that this `ErrorPage` style is a bit risky in how it's shared between so many components. I would like to see [this fixed](https://github.com/mozilla/addons-frontend/issues/8873) but as a quicker fix, I suggest at least renaming the class name from `ErrorPage` to `Errors` since that follows the structure of how it was before you moved the file. Just be careful to rename all of the other instances of `ErrorPage`.